components/eam/src/physics/rrtmgp/radiation.F90

```diff
@@ -467,33 +471,23 @@ subroutine get_aerosol_optical_property_from_file(month_int,aerosol_optical_prop
 
       !assign base_file_name the name of the CCN file being used 
       base_file_name = "spa_optics_file_"
-      optics_lookup_dir = "/p/lustre2/beydoun1/aerosol_optics"
 
       mon_str = adjustl(mon_str)
 
      !retrieve the name of the relevant file by combining base file name !with
      !!month and full file path:
-      filename = trim(optics_lookup_dir)//'/'//trim(base_file_name)//trim(mon_str)//'.nc'
-
-      write(iulog,*) 'file name is', filename
+!      filename = trim(optics_lookup_dir)//'/'//trim(base_file_name)//trim(mon_str)//'.nc'
```

Comment:
Might as well delete this commented out line.

Reply:
deleted. Missed it during last cleanup.